### PR TITLE
added avelon wisely carbon sense

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/avelon/wiselyCarbonSense/uplink.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+var Bits = require('bits');
+
+function consume(event) {
+  var payload = event.data.payload_hex;
+  var bits = Bits.hexToBits(payload);
+  var data = {};
+  var date = new Date();
+
+  emit('sample', { data: { "statusPercent": 100.0 * Bits.bitsToUnsigned(bits.substr(0, 8)) / 254.0 }, topic: "lifecycle" });
```
Comment (mariusgiger) [inline types/avelon/wiselyCarbonSense/uplink.js:9]: assign statusPercent to variable before that

File: types/avelon/wiselyCarbonSense/uplink.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    var e = Bits.bitsToUnsigned(bits.substr(pointer, 8)) << 8; pointer += 8;
+    var f = Bits.bitsToUnsigned(bits.substr(pointer, 8)); pointer += 8;
+
+    data.co2 = (e | f);
+
+    if (i == 0) {
+      emit('sample', { data: data });
+    } else {
+      var outTime = new Date(date.setMinutes(date.getMinutes() - 10));
+      emit('sample', { data: data, time: outTime });
```
Comment (mariusgiger) [inline types/avelon/wiselyCarbonSense/uplink.js:32]: what does time do?

**Review by mariusgiger — CHANGES_REQUESTED** [review]
some minor comments

File: types/avelon/wiselyCarbonSense/uplink.js (hunk 2 above)
Comment (J2thatsme) [reply types/avelon/wiselyCarbonSense/uplink.js:32]: Its possible that multiple datapoints are sent. So time changes what time the datapoint is saved on the database

Comment (mariusgiger) [reply types/avelon/wiselyCarbonSense/uplink.js:32]: @Biglr how to set the time from the script

Comment (Biglr) [reply types/avelon/wiselyCarbonSense/uplink.js:33]: @mariusgiger it should be possible by setting a JS date object in the `timestamp` property of the object.
The `time` property is from the old script running stuff, so please remove that wherever it is used @J2thatsme . If that doesn't do the trick, please let me know and I'll do some investigating.

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**